man/systemd-sleep.conf.xml

```diff
@@ -162,6 +160,9 @@
         url="https://www.kernel.org/doc/html/latest/admin-guide/pm/sleep-states.html#basic-sysfs-interfaces-for-system-suspend-and-hibernation">the
         kernel documentation</ulink> for more details.</para>
 
+        <para>Note that hybrid sleep corresponds to the <literal>suspend</literal> disk mode. If <varname>HybridSleepMode=</varname>
+        is overridden, you might get plain hibernation instead.</para>
```

Comment:
that smells as if we shouldn't allow setting HybridSleepMode= either

Reply:
-> #29688